pom.xml

```diff
@@ -63,6 +63,12 @@
       <version>10.50.2</version>
     </dependency>
 
+    <dependency>
```

Comment:
Not that I'm for heavily depending on a parent pom for dependency versioning etc but it's already defined in the parent https://github.com/ONSdigital/rm-common-config/blob/master/pom.xml#L62

Reply:
Agreed offline that we can do this another time as it doesn't have any impact